shiftleft-python-demo/flask_webgoat/__init__.py

```
@@ -0,0 +1,34 @@
+    app.secret_key = os.urandom(24)  # PRECOGS_FIX: use a random secret key
+
+    db_path = Path(DB_FILENAME)
+    if db_path.exists():
+        db_path.unlink()
+
+    conn = sqlite3.connect(DB_FILENAME)
+    create_table_query = """CREATE TABLE IF NOT EXISTS user
+    (id INTEGER PRIMARY KEY, username TEXT, password TEXT, access_level INTEGER)"""
+    conn.execute(create_table_query)
+
+    hashed_password = generate_password_hash('maximumentropy')  # PRECOGS_FIX: store hashed password
```

Ruff (0.15.6)

[error] Undefined name `generate_password_hash`

(F821)